src/paperqa/types.py

```diff
@@ -358,7 +358,7 @@ def filter_content_for_user(self) -> None:
                     # Similar to the explanation in `map_fxn_summary`'s internals
                     # on why we drop embeddings, drop embeddings here too because
                     # embeddings aren't displayed to front end users
-                    doc=c.text.doc.model_dump(exclude={"embedding"}),
+                    doc=c.text.doc.model_dump(exclude={"embedding", "other"}),
```

Comment:
Do you mind adding a code comment saying why we drop `other`

Also, should we do the same thing in `core.map_fxn_summary`?